odoo/tools/misc.py

```diff
@@ -7,6 +7,7 @@
 """
 from functools import wraps
 import babel
+import babel.dates
```

Comment:
grep other files using babel.dates without importing?
Seems like it's covered? e.g. http://runbot23.odoo.com/runbot/static/build/709426-master-16e690/coverage/odoo_addons_sale_timesheet_models_project_overview_py.html#n245

Reply:
In fact, this import automagically fixes missing imports of `babel.dates` everywhere else in addons, because it happens very early when we `import odoo`. It's better to import it explicitly in addons, but it's not strictly required.
That makes the change of Babel version safe enough for 13.0.